FHEM/00_SIGNALduino.pm

```diff
@@ -270,7 +270,7 @@ my %matchListSIGNALduino = (
 my %symbol_map = (one => 1 , zero =>0 ,sync => '', float=> 'F', 'start' => '');
 
 ## rfmode for attrib & supported rfmodes
-my @rfmode = ('KOPP_FC','Lacrosse_mode1','Lacrosse_mode2','Lacrosse_mode4','PCA301','SlowRF');
+my @rfmode = ('Bresser_5in1','KOPP_FC','Lacrosse_mode1','Lacrosse_mode2','Lacrosse_mode4','PCA301','SlowRF');
```

Comment:
War die Erkenntnis nicht, dass die RF Modes sich auf mehrere Sensoren / Hersteller beziehen können?

Reply:
Wir brauchen den rfmode aber z.B. trotzdem noch in der sub SIGNALduino_Attr_rfmode um die richtigen Register-Settings zum SIGNALduino zu schicken.